config/webpack.config.dev.js

```diff
@@ -0,0 +1,275 @@
+'use strict';
```

Comment:
I assume everything above this file was generated from `eject`

Reply:
Yep, the only files I manually changed were this one, `webpack.config.prod.js`, `index.html` (for the favicon), `App.js`,  and `App.scss`.